python/helper-image/Dockerfile

```diff
@@ -14,7 +14,9 @@
 
 # This Dockerfile creates a debug helper base image for Python.
 # It provides installations of debugpy, ptvsd, pydevd, and pydevd-pycharm
-# for Python 2.7, 3.7, 3.8, and 3.9.
+# for Python 2.7, 3.5, 3.6, 3.7, 3.8, and 3.9.
+#   - Apache Beam is based around Python 3.5
```

Comment:
nit: not sure these are necessary, maybe in container-debug-support README or skaffold README you could have supported python versions listed (or popular products that have a supported python version - eg: Apache Beam, etc.)

Reply:
I thought this was the better location to avoid future contributors inadvertently removing older versions.  End users likely will never find this repository!

Reply:
But I will update the versions in the README.